<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="choose_time_spinBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>30</number>
    </property>
    <property name="maximum">
     <number>600</number>
    </property>
    <property name="singleStep">
     <number>15</number>
    </property>
   </widget>
   <widget class="QLabel" name="choose_time">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите время</string>
    </property>
   </widget>
   <widget class="QPushButton" name="play">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>170</y>
      <width>150</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Играть</string>
    </property>
   </widget>
   <widget class="QLabel" name="itogo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Итого:</string>
    </property>
   </widget>
   <widget class="QLabel" name="price">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="count_price">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>140</y>
      <width>150</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Рассчитать стоимость</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
